tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/ServiceRecipients GetAll.feature

```diff
@@ -1,4 +1,4 @@
-﻿Feature: display all of the selected service recipients for an order
+﻿Feature: display all of the service recipients for an order
```

Comment:
I think we should also update the description of the first scenario to `Scenario: get the service recipients from an existing order`.

Reply:
Updated Scenario description with PR [435](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering/pull/435)